test/e2e/managedcluster_test.go

```diff
@@ -25,5 +31,129 @@ var _ = ginkgo.Describe("Testing ManagedCluster", func() {
 				return nil
 			}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("Get CA from apiserver", func() {
```

Comment:
I think it will be better to start a new Context to include your ca cases, not in "Get ManagedCluster cpu worker capacity" Context.

Reply:
done